tests/lifecycle/tests/lifecycle_deployment_scaling.go

```diff
@@ -15,8 +15,11 @@ import (
 var _ = Describe("lifecycle lifecycle-deployment-scaling", func() {
 
 	BeforeEach(func() {
+		_, err := lifehelper.ValidateClusterIsStable()
```

Comment:
So it doesn't make sense that ValidateClusterIsStable returns anything else than the err value. Also, another minor suggestion: a better name for that function could be "WaitUntilClusterIsStable".